source/common/event/real_time_system.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "envoy/event/timer.h"
+
+#include "common/common/utility.h"
+
+namespace Envoy {
+namespace Event {
+
+/**
+ * Real-time implementation of TimeSystem.
```

Comment:
Same comment here - `RealTimeSystem` reads OK, but "Real-time" makes it sound like it's some extra high resolution thing. How about "Real clock implementation of TimeSystem"?

Reply:
Changed to "Real-world time"